pkgs/misc/vim-plugins/default.nix

```diff
@@ -2,7 +2,7 @@
 { fetchurl, bash, stdenv, python, go, cmake, vim, vimUtils, perl, ruby, unzip
 , which, fetchgit, fetchFromGitHub, fetchhg, fetchzip, llvmPackages, zip
 , vim_configurable, vimPlugins, xkb_switch, git
-, Cocoa
+, Cocoa ? null
```

Comment:
I was testing (and I'm testing it everytime I update vim plugins) it myself 
and it works.
Can you give me more information about your problem?

-- 
Sincerely,
Arseniy Seroka



On 2 March 2016 14:20:24 Stefan Junker <notifications@github.com> wrote:

>> @@ -2,7 +2,7 @@
>>  { fetchurl, bash, stdenv, python, go, cmake, vim, vimUtils, perl, ruby, unzip
>>  , which, fetchgit, fetchFromGitHub, fetchhg, fetchzip, llvmPackages, zip
>>  , vim_configurable, vimPlugins, xkb_switch, git
>> -, Cocoa
>> +, Cocoa ? null
>
> It causes build failures on non-darwin, at least for me. It needs handling 
> in the outer scope.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/NixOS/nixpkgs/pull/13575/files#r54710553